### PR TITLE
[back] feat: Generate and tweet montly Top contributor figure

### PR DESCRIPTION
**Pull request by aidanjungo**
- [x] modif exiting test
- [x] add test for `tweet_top_contributor_graph`
- [x] add a django management command

### CODE REVIEW

**Comment by GresilleSiffle** [comment]
Before merging:

We could move the Tournesol logo in a resources folder in the `twitterbot`. We already have a logo in the `tournesol` app, should we re-use it?

We should precisely set the `matplotlib` version in the requirements to keep our build deterministic

**Comment by aidanjungo** [comment]
> We could move the Tournesol logo in a resources folder in the `twitterbot`. We already have a logo in the `tournesol` app, should we re-use it?

Yes it is in my TODO to move it, but I think i added it because we did not have it yet with the good format. I will check.


> We should precisely set the `matplotlib` version in the requirements to keep our build deterministic

Done

File: backend/twitterbot/settings.py [hunk 1]
```diff
@@ -27,6 +27,28 @@
     ),
 }
 
+# Monthly top contributor text template
+graph_title_text_template = {
+    "en": ("Who compared the most on Tournesol in {month_name} {year}"),
+    "fr": ("Qui a le plus comparé sur Tournesol en {month_name} {year}"),
```
Comment (amatissart) [inline backend/twitterbot/settings.py:33]:
```suggestion
    "en": ("Who compared the most on Tournesol in {month_name} {year}?"),
    "fr": ("Qui a le plus comparé sur Tournesol en {month_name} {year} ?"),
```

File: backend/twitterbot/management/commands/tweet_top_contribors.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    help = "Runs Tournesol Twitter Bot to tweet Top contributor figure"
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "-n",
+            "--bot-name",
+            type=str,
+            help="Name of the bot '@TournesolBot' or '@TournesolBotFR'",
+        )
```
Comment (amatissart) [inline backend/twitterbot/management/commands/tweet_top_contribors.py:16]: Would make sense to make this argument required

```suggestion
        parser.add_argument(
            "-n",
            "--bot-name",
            type=str,
            required=True,
            help="Name of the bot '@TournesolBot' or '@TournesolBotFR'",
        )
```

**Review by amatissart — APPROVED** [review]

**Pull request merged**